docs/backend/service-objects.md

```diff
@@ -4,20 +4,30 @@ sidebar_position: 20
 
 # Service Objects
 
-:::important
+Service objects are Plain Old Ruby Objects (POROs) which encapsulate a whole
+business process/user interaction. We rely on these objects in our codebase to remove complexity from the Models and Controller.
 
-We’re currently making rapid changes to the product so our docs may be out of date. If you need help, please email [yo@forem.com](mailto:yo@forem.com).
+Our services are located in `app/services` with the corresponding specs in
+`spec/services`.
 
-:::
+## Naming Conventions for Service Objects
 
-## What are Service Objects
+- We prefer to namespace our Service Objects especially when we potentially see the addition of more services under that namespace. This is adviseable rather than cluttering the root folder.
+- We namespace services in a plural form to avoid running into problems with Rails naming conventions. For example `Reactions::` instead of `Reaction::`.
+- To distinguish services from models we often follow the VerbNoun naming convention for services, like `HomePage::FetchArticles` instead of `HomePage::ArticleRetrieval`. However, sometimes the naming of the service may not fit within this convention and if you can provide strong reasons on why you should use a different name then we will leave it to your discretion.
```

Comment:
```suggestion
- To distinguish services from models we often follow the VerbNoun naming convention for services, like `HomePage::FetchArticles` instead of `HomePage::ArticleRetrieval`. However, sometimes the naming of the service may not fit within this convention and if you can rationalize about why you should use a different name then we will leave it to your discretion.
```